docker-compose.yml

```diff
@@ -1,21 +1,24 @@
 mongodb:
-  image: mongo:latest
-  container_name: mongodb
-  command: mongod --smallfiles
-  ports:
-   - "27017:27017"
-  volumes:
-   - ./mongo:/data/db:rw
-  volume_driver: local
+  image: bitnami/mongodb:latest
```

Comment:
Why did you change this to a Bitnami image over the official MongoDB image? What is the value of that change?